smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -197,7 +231,7 @@ contract BridgeBank is BankStorage, CosmosBank, EthereumBank, WhiteList {
         bytes memory _recipient,
         address _token,
         uint256 _amount
-    ) public payable onlyWhiteList(_token) {
+    ) public payable onlyEthTokenWhiteList(_token) validSifAddress(_recipient) {
         string memory symbol;
```

Comment:
There is an ethereum token whitelist and a cosmos token whitelist. This keeps the concerns separate and avoids attack vectors where a user could try to burn a whitelisted token